content/ios/query.md

```diff
@@ -106,13 +98,43 @@ SKYDatabase *privateDB = database;
 }];
 ```
 
-## Limiting and Offset
+## Complex Queries
+
+### Sorting the records
+We can sort records returned like this:
+
+```obj-c
+NSSortDescriptor *sortDescriptor = [NSSortDescriptor sortDescriptorWithKey:@"_updated_at" ascending:NO];     // sorted by modificationDate
+query.sortDescriptors = @[sortDescriptor];     // apply the NSSortDescriptor to the query
+```
+
+`SKYQuery` utilizes `NSPredicate` to apply filtering on query results. You can use other parameters to sort your quries.
+
+### Limiting and Offset
```

Comment:
I think `Pagination` is a better title, since we will also include `query.page`, etc. in this section